src/Tasks/ProcessJobQueueTask.php

```diff
@@ -43,6 +47,26 @@ public function run($request)
 
         $service = $this->getService();
 
+        // Ensure that log messages are visible when executing this task on CLI.
+        // TODO Replace with BuildTask logger: https://github.com/silverstripe/silverstripe-framework/issues/9183
```

Comment:
Please don't include TODO's in PR's, already enough of those in the codebase =)  

Better of linking to this PR:line in that issue

Reply:
If you feel strongly about this then please change the TODO tag in a commit. I don't ;)

Reply:
Sorry, I was cranky before - I've removed the TODO tag, and converted it to a standard comment referencing this context instead. Thanks again for following up on those PRs, and getting into the details during review!